samples/wifi/sta/src/main.c

```diff
@@ -24,6 +24,9 @@ LOG_MODULE_REGISTER(sta, CONFIG_LOG_DEFAULT_LEVEL);
 #include <zephyr/net/net_event.h>
 #include <zephyr/drivers/gpio.h>
 
+#include <net/wifi_credentials.h>
```

Comment:
Is this needed? As we are only calling the Wi-Fi mgmt ext API?

Reply:
Not needed.